nuitka/OptionParsing.py

```diff
@@ -667,14 +667,14 @@ def _handleHelpModes():
 
 execute_group.add_option(
     "--debugger",
-    "--gdb",
-    action="store_true",
+    "--gdb",  # Remove this now that this supports any arbitrary debugger?
     dest="debugger",
-    default=False,
+    default=None,
     help="""\
 Execute inside a debugger, e.g. "gdb" or "lldb" to automatically get a stack trace. The
 debugger is automatically chosen unless specified by name with the NUITKA_DEBUGGER_CHOICE
-environment variable. Defaults to off.""",
+environment variable or this flag. Defaults to off.""",
+    implicit_default="gdb",
```

Comment:
So what you did here was to make `--debugger` have an ability to accept arguments optionally? I am not so sure I like that idea, and it's not documented here how to do it. The argparse also doesn't have that, right?

And I couldn't say --debugger=lldb without repeating Nuitka code to make the automatic traceback, right? I added the option to be able to forget those.

I am not so sure if I wouldn't always prefer a separate option to provide those things, like --debugger-args and want sane defaults to be commony used, even and esp. for valgrind memcheck.